site/docs/master/upgrade-to-1.4.md

```diff
@@ -23,23 +24,29 @@ If you're not yet running at least Velero v1.2, see the following:
 
     ```bash
     Client:
-        Version: v1.3.2
+        Version: v1.4.0-beta.1
         Git commit: <git SHA>
     ```
 
 1. Update the container image used by the Velero deployment and, optionally, the restic daemon set:
 
     ```bash
     kubectl set image deployment/velero \
-        velero=velero/velero:v1.3.2 \
+        velero=velero/velero:v1.4.0-beta.1 \
         --namespace velero
 
     # optional, if using the restic daemon set
     kubectl set image daemonset/restic \
-        restic=velero/velero:v1.3.2 \
+        restic=velero/velero:v1.4.0-beta.1 \
         --namespace velero
     ```
 
+1. Update the Velero custom resource definitions (CRDs) to include the new backup progress fields:
```

Comment:
Should this step be before updating the container image?

Reply:
eh, shouldn't really matter one way or another - the updated image will run just fine without the CRD updates, and if the CRDs aren't updated, backups just won't get progress updates.

Reply:
As long as the pod doesn't required to be restarted for the new CRDs to be picked up it's fine.